src/main/scala/fastq.scala

```diff
@@ -269,7 +269,7 @@ case object fastq {
 
     def parseFastqPhred33: Iterator[Option[FASTQ]] =
       lines
-        .grouped(4)
+        .grouped(4).filter(_.size == 4) // grouped is broken
```

Comment:
> Sometimes, reading the documentation is more useful than reading old discussions about somebody's wrong decisions.

As I said here https://github.com/ohnosequences/fastarious/pull/13#discussion_r102762561 this is just a hack for `GroupedIterator`, and the behavior will be the broken one in all other cases where you don't work with an iterator (see for example [this](http://stackoverflow.com/a/7959167/614394)). Documenting that something is broken only (if anything) adds to the problem, presenting it as standard, correct behavior. I did read the docs, and the source code of `Iterator`; you can see there that in terms of performance there will be (virtually) no difference between `withPartial(false)` and filtering. 

Given all this, and particularly the fact that for other collections grouped/sliding/whatever is broken  too and has no `withPartial` hack, it is much more sensible defaulting to filter by size.

Hope this helps.